cmd/ef_tests/blockchain/test_runner.rs

```diff
@@ -33,7 +33,7 @@ pub fn run_ef_test(test_key: &str, test: &TestUnit) {
         let hash = block.hash();
 
         // Attempt to add the block as the head of the chain
-        let chain_result = add_block(block, &store);
+        let chain_result = Blockchain::default().add_block(block, &store);
```

Comment:
The idea is that the blockchain will be stateful, 

let blockchain = Blockchain:new(evm, store, <other configs>);
for block in blocks {
   blockchain.add_block(block)
}

Otherwise a blockchain struct wouldn't make much sense